diff --git a/cli.py b/cli.py
index 016f294..452a954 100644
--- a/cli.py
+++ b/cli.py
@@ -15,7 +15,7 @@ import vision.track.interpolation
 import turkic.models
 from models import *
 import cStringIO
-import Image, ImageDraw, ImageFont
+from PIL import Image, ImageDraw, ImageFont
 import qa
 import merge
 import parsedatetime
@@ -501,7 +501,7 @@ class visualize(DumpCommand):
         offset = 100
         for im, frame in frames:
             aug = Image.new(im.mode, (im.size[0], im.size[1] + offset))
-            aug.paste("black")
+            #aug.paste("black")
             aug.paste(im, (0, 0))
             draw = ImageDraw.ImageDraw(aug)
 
diff --git a/models.py b/models.py
index 4492de1..7520607 100644
--- a/models.py
+++ b/models.py
@@ -3,7 +3,7 @@ import turkic.models
 from sqlalchemy import Column, Integer, Float, String, Boolean, Text
 from sqlalchemy import ForeignKey, Table, PickleType
 from sqlalchemy.orm import relationship, backref
-import Image
+from PIL import Image
 import vision
 from vision.track.interpolation import LinearFill
 import random
